src/sections/body/body.section.js

```diff
@@ -41,28 +41,25 @@ const Body = () => {
     }
   })
 
-  const [showAltForm, toggleAlt] = useState({ showAlt:false, meal:'', day:'', content:'' })
-
-  const toggleShowAltForm = (meal, day, content, bool) => {
-    toggleAlt({
-      showAlt: bool ? true : false,
-      meal:meal, day:day, content:content
-    });
-  };
-
+  const [altForm, setAltForm] = useState({ showAlt:false, meal:'', day:'', content:'' })
+  const [altShow, setAltShow] = useState(false)
```

Comment:
`altShow` is vague as well, it's hard to tell what this state is for by just reading this line. Can you come up with something else with enough context?